### PR TITLE
 sign in page- fixed

### PR DESCRIPTION
**Pull request by erezdav22**
the login isn't working yet, just showing me the login button but no forrm to fill, does anyone know what is missing?

### CODE REVIEW

File: ultimatejob/urls.py [hunk 1]
```diff
@@ -23,8 +23,9 @@
     path('signup/', views.sign_up, name="signup"),
     path('personal_area/', views.personal_area, name="personal_area"),
     path('sign_in/', views.sign_in, name="sign_in"),
+    path('sign_in/', views.sign_in_view, name="sign_in"),
```
Comment (simzacks) [inline ultimatejob/urls.py:25]: You have 2 paths for sign_in here. I don't know which one it will use

File: ultimatejob/urls.py [hunk 2]
```diff
@@ -23,8 +23,9 @@
...
     path('available_jobs/', views.available_jobs, name="available_jobs"),
     path('accounts/', include('django.contrib.auth.urls')),
-    path('signin/', views.signup_view, name="sign_in")
+   # path('signin/', views.signup_view, name="sign_in")
```
Comment (simzacks) [inline ultimatejob/urls.py]: remove code you aren't using. Don't leave it as a comment

File: ultimatejobweb/templates/sign_in.html [hunk 3]
```diff
@@ -56,17 +56,19 @@ <h1 class="display-3">UltimateJob</h1>
...
-            <input type="email" class="form-control" id="inputEmail4" placeholder="Email">
-          </div>
-          <div class="form-group col-md-6">
-            <label for="inputPassword4">Password</label>
-            <input type="password" class="form-control" id="inputPassword4" placeholder="Choose Password">
-      
-            <a href="/personal_area"  class="btn btn-outline-success">sign in</a>
-      </form>
+ 
+<!-- <form class="site-form" action="{% url 'sign_in'%}" method="post">
```
Comment (simzacks) [inline ultimatejobweb/templates/sign_in.html]: Remove coffee that you're not using. Don't just comment it out

**Review by simzacks — CHANGES_REQUESTED** [review]
Minor not about removing commented out code. Other than that it looks good

File: ultimatejobweb/templates/personal_area.html [hunk 4]
```diff
@@ -71,6 +77,7 @@ <h1 class="display-3">UltimateJob</h1>
     </div>
 </div>
 
+<footer>
```
Comment (simzacks) [inline ultimatejobweb/templates/personal_area.html]: should that be /footer?

File: ultimatejobweb/templates/home_page.html [hunk 5]
```diff
@@ -76,6 +82,8 @@ <h1 class="display-3">Welcome to UltimateJob!</h1>
     </div>
 </div>
 
+<footer>
```
Comment (simzacks) [inline ultimatejobweb/templates/home_page.html]: should that be /footer?

File: ultimatejobweb/templates/available_jobs.html [hunk 6]
```diff
@@ -109,6 +116,7 @@ <h1 class="display-3">UltimateJob</h1>
     </div>
 </div>
 
+<footer>
```
Comment (simzacks) [inline ultimatejobweb/templates/available_jobs.html]: should that be /footer?

File: ultimatejobweb/templates/sign_in.html [hunk 7]
```diff
@@ -82,8 +83,7 @@ <h1 class="display-3">UltimateJob</h1>
     </div>
 </div>
 
-
-
+<footer>
```
Comment (simzacks) [inline ultimatejobweb/templates/sign_in.html]: should that be /footer?

File: ultimatejobweb/templates/sign_up.html [hunk 8]
```diff
@@ -83,6 +88,7 @@ <h1 class="display-3">UltimateJob</h1>
     </div>
 </div>
 
+<footer>
```
Comment (simzacks) [inline ultimatejobweb/templates/sign_up.html]: should that be /footer?

Comment (erezdav22) [reply ultimatejobweb/templates/sign_up.html]: Yes, i used the footer tag and added a css file to make the footer stay at the bottom of the page, at the same spot in all pages

Comment (erezdav22) [reply ultimatejobweb/templates/sign_up.html]: I see what you mean now, yes i didn't add the / to close the footer tag, silly mistake..

**Review by simzacks — APPROVED** [review]

**Pull request merged**